presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/AbstractResourceConfigurationManager.java

```diff
@@ -177,13 +177,15 @@ protected void configureGroup(ResourceGroup group, ResourceGroupSpec match)
         }
         group.setMaxQueuedQueries(match.getMaxQueued());
         group.setMaxRunningQueries(match.getMaxRunning());
+        match.getQueuedTimeout().ifPresent(timeout -> group.setQueuedTimeLimit(timeout));
+        match.getRunningTimeout().ifPresent(timeout -> group.setRunningTimeLimit(timeout));
         if (match.getSchedulingPolicy().isPresent()) {
             group.setSchedulingPolicy(match.getSchedulingPolicy().get());
         }
         if (match.getSchedulingWeight().isPresent()) {
             group.setSchedulingWeight(match.getSchedulingWeight().get());
         }
-        if (match.getJmxExport().isPresent()) {
+        if (match.getJmxExport().isPresent() && match.getJmxExport().get() != group.getJmxExport()) {
```

Comment:
Without that an exception is thrown if the previous and new values are the same. I'll add a comment.